src/__tests__/ToastHeader.spec.js

```diff
@@ -1,44 +1,30 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import { ToastHeader } from '..';
+import { testForCustomClass, testForDefaultClass } from '../testUtils';
 
 describe('ToastHeader', () => {
   it('should render with "toast-header" class', () => {
-    const wrapper = shallow(<ToastHeader>Yo!</ToastHeader>);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('toast-header')).toBe(true);
+    testForDefaultClass(ToastHeader, 'toast-header');
   });
 
   it('should render additional classes', () => {
-    const wrapper = shallow(<ToastHeader className="other">Yo!</ToastHeader>);
-
-    expect(wrapper.hasClass('other')).toBe(true);
-    expect(wrapper.hasClass('toast-header')).toBe(true);
+    testForCustomClass(ToastHeader);
   });
 
   it('should render close button', () => {
-    const wrapper = shallow(
-      <ToastHeader toggle={() => {}} className="other">
-        Yo!
-      </ToastHeader>,
-    );
+    render(<ToastHeader toggle={() => {}}>Yo!</ToastHeader>);
 
-    expect(wrapper.hasClass('other')).toBe(true);
-    expect(wrapper.hasClass('toast-header')).toBe(true);
-    expect(wrapper.find('button.btn-close').length).toBe(1);
+    expect(screen.getByLabelText(/close/i)).toBeInTheDocument();
   });
 
   it('should render custom tag', () => {
-    const wrapper = shallow(<ToastHeader tag="p">Yo!</ToastHeader>).childAt(0);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.type()).toBe('p');
+    render(<ToastHeader tag="p">Yo!</ToastHeader>);
```

Comment:
I believe you made a `testForCustomTag` utility function. So maybe use that?

Reply:
`TestHeader` works a bit differently. The `prop` name for changing the tag for external class in `ToastHeader` is `wrapTag` and not `tag`. So can't use that function here.

Reply:
Makes sense and sounds good!